src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -0,0 +1,710 @@
+            CHAMPVA recipients, or if you have any other questions, please call
+            our enrollment case management team at 1-877-222-VETS (
+            <a className="help-phone-number-link" href="tel:1-877-222-8387">
+              1-877-222-8387
+            </a>
+            ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+          </p>
+        </>
+      );
+      break;
+
+    case HCA_ENROLLMENT_STATUSES.activeDuty:
+    case HCA_ENROLLMENT_STATUSES.canceledDeclined:
+    case HCA_ENROLLMENT_STATUSES.closed:
+    case HCA_ENROLLMENT_STATUSES.ineligFugitiveFelon:
+    case HCA_ENROLLMENT_STATUSES.ineligMedicare:
+    case HCA_ENROLLMENT_STATUSES.ineligOther:
+    case HCA_ENROLLMENT_STATUSES.ineligOver65:
+    case HCA_ENROLLMENT_STATUSES.ineligRefusedCopay:
+    case HCA_ENROLLMENT_STATUSES.rejectedIncWrongEntry:
+    case HCA_ENROLLMENT_STATUSES.rejectedRightEntry:
+    case HCA_ENROLLMENT_STATUSES.rejectedScWrongEntry:
+      content = (
+        <>
+          <h4>What should I do if I have questions about my eligibility?</h4>
+          <p>
+            Please call our enrollment case management team at 1-877-222-VETS (
+            <a className="help-phone-number-link" href="tel:1-877-222-8387">
+              1-877-222-8387
+            </a>
+            ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+          </p>
+        </>
+      );
+      break;
+
+    case HCA_ENROLLMENT_STATUSES.pendingMt:
+    case HCA_ENROLLMENT_STATUSES.pendingPurpleHeart:
+      content = (
+        <>
+          <h4>How do I submit this information to VA?</h4>
+          <p>
+            Please call our enrollment case management team at 1-877-222-VETS (
+            <a className="help-phone-number-link" href="tel:1-877-222-8387">
+              1-877-222-8387
+            </a>
+            ) for directions on how to submit your information. We’re here
+            Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+          </p>
+        </>
+      );
+      break;
+
+    case HCA_ENROLLMENT_STATUSES.pendingOther:
+    case HCA_ENROLLMENT_STATUSES.pendingUnverified:
+      content = (
+        <>
+          <h4>
+            How will I know if VA needs more information from me to verify my
+            military service?
+          </h4>
+          <p>
+            If we need more information, we'll send you a letter in the mail. If
+            you have any questions, please call our enrollment case management
+            team at 1-877-222-VETS (
+            <a className="help-phone-number-link" href="tel:1-877-222-8387">
+              1-877-222-8387
+            </a>
+            ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+          </p>
+        </>
+      );
+      break;
+
+    default:
+      break;
+  }
+  return content;
+}
+
+// This block is _only_ shown if the user does not qualify due to the character
+// of their discharge
+export function getFAQBlock2(enrollmentStatus) {
```

Comment:
I've only glanced at this but this might be able to be consolidated into a single array and generated with `filter`. Each item in the array could have an object with 
``` 
const NamedJSXFragment = (<> <blurb /> </>);

[...,
{
content: (<NamedJSXFragment / >),
statuses: [ HCA_ENROLLMENT_STATUS.ineligCharacterOfDischarge, 
HCA_ENROLLMENT_STATUS.ineligCitizens ...]
},
...]
```

then you could filter the content blocks by with `status.find(enrollmentStatus)` as long as there isn't any complicated ordering logic with the blurbs. 

Basically thinking of it like an db of content blurbs with statuses associated with them that's queried with filter- I assume this would work because only one attribute is being considered `enrollmentStatus`.

Reply:
I'm sure I'm missing something, but I don't the benefit of refactoring this how you're proposing. Are you saying to just refactor the internals of the `getFAQBlock()` functions? Or doing the refactor at a higher level?

Reply:
Yeah- instead of having `Block1` `Block2` functions just putting it into a single array and filtering on it. The main advantage is it's a little more declarative to explicitly show how the blurbs are associated with the status codes with a data structure instead of implicitly with a switch statement.

Reply:
I'll look at trying that. I honestly won't be able to tell if it'll make it clearer or not until I see it in action.